benchmarking/training_scripts/resnet_cifar10/resnet_cifar10.py

```diff
@@ -178,7 +165,13 @@ def objective(config):
         sampler=valid_sampler,
         pin_memory=True,
     )
+    return train_loader, valid_loader
 
+
+def _create_training_objects(config):
+    num_gpus = config.get("num_gpus")
+    if num_gpus is None:
+        num_gpus = 1
```

Comment:
What are the possible scenarios where `num_gpus is None`?

Reply:
If the parameter is not given. We could deal with this in argparse, just saw that